src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started his work!");

```

Comment:
The message 'Truck started his work!' is not informative enough. According to the checklist, the message should indicate what type of `Machine` is working. Consider specifying that it is a `Truck` that has started working, for example: 'Truck is now working.'